querybook/server/datasources/board.py

```diff
@@ -135,6 +145,10 @@ def add_board_item(board_id, item_type, item_id):
             is None,
             "Item already exists",
         )
+        api_assert(
```

Comment:
just realized this check can be done without any querying, can you put it after line 125